Mathlib/Analysis/NormedSpace/LogDeriv.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+import Mathlib.Analysis.Complex.LocallyUniformLimit
+import Mathlib.Analysis.Calculus.LogDeriv
+import Mathlib.Analysis.SpecialFunctions.Complex.LogDeriv
+
+/-!
+# Logarithmic Derivatives
+
+
+We prove some facts about logarithmid derivatives and limits of logarithmic deriviative, for example
+that the logarithmic derivative of a sequence of functions converging
+locally uniformly to a function is the logarithmic derivative of the limit function. This is useful
+for example for the Mittag-Leffler expansion of the cotangent function.
+-/
+
+noncomputable section
+
+open Filter Function
+
+open scoped Topology BigOperators Classical
+
+section log
+
+/-- The derivative of `log ∘ f` is the logarithmic derivative provided `f` is differentiable and
+we are on the slitPlane. -/
+lemma Complex.deriv_log_comp_eq_logDeriv {f : ℂ → ℂ} {x : ℂ} (h₁ : DifferentiableAt ℂ f x)
+    (h₂ : f x ∈ Complex.slitPlane) : deriv (Complex.log ∘ f) x = logDeriv f x := by
+  have A := (HasDerivAt.clog h₁.hasDerivAt h₂).deriv
+  rw [← h₁.hasDerivAt.deriv] at A
+  simp only [logDeriv, Pi.div_apply, ← A, Function.comp_def]
```

Comment:
Please move this and the next lemma to the files, where `HasDerivAt.clog` and `deriv.log` come from.